.github/workflows/source_clear_cron.yml

```diff
@@ -12,7 +14,13 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v3
+      - uses: actions/setup-go@v3
+        with:
+          go-version: '1.21.0'
+          check-latest: true
       - name: Source clear scan
         env:
           SRCCLR_API_TOKEN: ${{ secrets.SRCCLR_API_TOKEN }}
-        run: curl -sSL https://download.sourceclear.com/ci.sh | bash -s - scan
+        run: |
```

Comment:
I wonder if we still need to scan with Source Clear, since Prisma Cloud is an implied part of the CI now 🤔